docs/spec/v1beta1/helmcharts.md

```diff
@@ -130,6 +140,26 @@ spec:
   interval: 5m
 ```
 
+Specify the values files to be when installing/upgrading the helm chart:
+
+```yaml
+apiVersion: source.toolkit.fluxcd.io/v1beta1
+kind: HelmChart
+metadata:
+  name: redis
+  namespace: default
+spec:
+  chart: redis
+  version: 10.5.7
+  sourceRef:
+    name: stable
+    kind: HelmRepository
+  interval: 5m
+  valuesFiles:
+    - values.yaml
+    - values-production.yaml
+```
```

Comment:
I added a specific example to the docs on how to use `valuesFiles`. I'm hoping that by explicitly including the `values.yaml` it will be clear that they are not included by default (in addition to documenting it directly on the Spec).

Should Redis be the example (the Redis chart does have a `values` and `values-production.yaml` but the `stable` repository is deprecated)? Should a `HelmRepository` be the example Source type? Should it be this far up in the list of examples?